packages/extension/app/background.js

```diff
@@ -46,6 +49,26 @@ const connect = function (host, path, extraOpts) {
     })
   })
 
+  const listenToOnBeforeHeaders = once(() => {
+    browser.webRequest.onBeforeSendHeaders.addListener((details) => {
+      if (
+        details.parentFrameId !== 0
```

Comment:
why must `parentFrameId` be `0`? it's not clear to me from the docs what this represents. is this `-1` for `window.top` and `0` for the iframe loaded from the aut iframe? is this because the aut iframe is created first, before the spec iframe is created, so it has the ID of `0`?

this could use a comment

Reply:
`parentFrameId === 0` means the parent frame is `top`, so the request belongs to an iframe at the top-level and not a nested iframe. I'll add a comment for this.

Reply:
Addressed in [`2471643`](https://github.com/cypress-io/cypress/pull/19323/commits/24716432da446f002771f79f0eb3ecb4ef56cf61)